src/CreateScenarioButton/components/CreateScenarioDialog/CreateScenarioDialog.js

```diff
@@ -228,8 +232,9 @@ const CreateScenarioDialog = ({
           <Grid item xs={12}>
             { isMaster || !currentScenarioSelected
               ? (<Autocomplete
+                  data-cy="dataset-select"
```

Comment:
Could be interesting to prefix the id by  `create-scenario-dialog` in order to narrow the possibilities to have several field with the same data-cy id?
Something like  `create-scenario-dialog-dataset-select`